### ISSUE TITLE
Lineageos21: Bitwarden cannot be used for passkey authentication on mobile

### ISSUE THREAD

**Issue by somova** [bitwarden/mobile#3299]
### Steps To Reproduce

1. Ensure Bitwarden mobile is set as provider for "Passwords, Passkeys and Data Services"
2. Open mobile Browser
3. Open webpage "https://webauthn.io/"
4. Try to register a passkey or authenticate with a passkey

### Expected Result

Android should offer to select the favorite passkey provider to use with the webpage

### Actual Result

Google only offers to store a new or use an already stored passkey in the Google Account or to select another device. The list of other devices only contains NFC, USB or other smartphone.

### Operating System

Android

### Operating System Version

LineageOS 21 (Build: 20240610; Android 14)

### Device

Oneplus 9 Pro (LE2123)

### Build Version

2024.5.1 (10574)

### Beta

- [ ] Using a pre-release version of the application.

**Comment by SergeantConfused** [comment]
Hi @somova,

Thank you for this report. Did you test this in Chrome after performing the steps that are [outlined in the documentation](https://bitwarden.com/help/auto-fill-android/#setup-bitwarden-for-use-with-passkeys)?

Thank you in advance,

**Comment by somova** [comment]
Sorry, I forgot to mention the tested apps:

- Vivaldi mobile browser (v.6.7.3335.149)
- Firefox mobile browser (v.127.0)
- Chrome mobile browser (v.126.0.6478.71)

Additionally, I followed the [documentation](https://bitwarden.com/help/auto-fill-android/#setup-bitwarden-for-use-with-passkeys) and enabled flag "M124", but no avail.



_Addition:_
At first I did not find the second option (Enabled for 3rd party passkey providers under Android Credential Management for passkeys). After a closer look I found and enabled it. Now, the Chrome browser shows a bitwarden prompt when authenticating at "webauthn.io", but throws the following error message:

`Fehler beim Lesen des Passkeys. Es gab ein Problem beim Lesen deines Passkeys für webauthn.io. Versuche es später erneut`

Since my LineageOS smartphone is rooted. Are there any log files I could inspect?

**Comment by micahblut** [comment]
@somova Do you know if LineageOS supports Google Play Services?

**Comment by msiqx9** [comment]
> @somova Do you know if LineageOS supports Google Play Services?

Yes, all my OnePlus phones with LineageOS 21 have Google Play Services installed.

**Comment by somova** [comment]
> @somova Do you know if LineageOS supports Google Play Services?

Yes, LineageOS supports Google Play Services. During initial installation phase of LineageOS the user can decide to add the Google addon package. On my smartphone Google Services including Google Play are available

**Comment by somova** [comment]
Now authentication succeeds with the Chrome browser. The error message

> Fehler beim Lesen des Passkeys. Es gab ein Problem beim Lesen deines Passkeys für webauthn.io. Versuche es später erneut

disappers if the session timeout in the bitwarden mobile app is set to a value other than "immediately". Registering a passkey is still not offered.
